dogowner/test_dog_owner.py

```diff
@@ -0,0 +1,97 @@
+from .models import DogOwner
+from django.contrib.auth.models import User
+from django.core.exceptions import ValidationError
+import pytest
+
+EMAIL = 'testuser@gmail.com'
+USERNAME = 'testuser01'
+PASSWORD = 'testpassowrd'
+DOG_NAME = 'kliford'
+FIRST_NAME = 'NEW'
+LAST_NAME = 'USER'
+PHONE_NUMBER = 1234567890
+DOG_RACE = 'lavrador'
+DOG_PICTURE_URL = 'https://www.google.com/'
+DOG_AGE = 10
+DOG_WEIGHT = 6
+DOG_GENDER = 'M'
+
+
+@pytest.fixture
+def dogOwner_01():
+    return DogOwner.create(email=EMAIL,
+                           username=USERNAME,
+                           password=PASSWORD,
+                           dog_name=DOG_NAME,
+                           first_name=FIRST_NAME,
+                           last_name=LAST_NAME,
+                           phone_number=PHONE_NUMBER,
+                           dog_race=DOG_RACE,
+                           dog_picture_url=DOG_PICTURE_URL,
+                           dog_age=DOG_AGE,
+                           dog_weight=DOG_WEIGHT,
+                           dog_gender=DOG_GENDER
+                           )
+
+
+@pytest.mark.django_db
+class TestDogOwnerModel:
+    def test_persist_dog_owner(self, dogOwner_01):
+        dogOwner_01.save()
```

Comment:
save() is called on creation, why is it necessary to call again?

Reply:
You right. Fixed .